pyquil/api/compiler.py

```diff
@@ -223,24 +223,33 @@ def _rb_sequence_payload(self, depth, qubits, gateset):
                    "gateset": [gate.out() for gate in gateset]}
         return payload
 
-    def generate_rb_sequence(self, depth, qubits, gateset):
+    def generate_rb_sequence(self, depth, gateset):
         """
-        Construct a randomized benchmarking experiment on the given qubits,
-        decomposing into gateset.
+        Construct a randomized benchmarking experiment on the given qubits, decomposing into gateset.
 
         The JSON payload that is parsed is a list of lists of indices, or Nones. In the
         former case, they are the index of the gate in the gateset.
 
-        :param int depth: The number of Clifford gates to include in the randomized benchmarking experiement.
-        :param int qubits: The number of qubits to generate a randomized benchmarking sequence for.
-        :param list gateset: A list of pyquil gates to decompose the Clifford elements into.
-        """
-        payload = self._rb_sequence_payload(depth, qubits, gateset)
+        :param int depth: The number of Clifford gates to include in the randomized benchmarking experiment. This is
+         different than the number of gates in the resulting experiment.
+        :param list gateset: A list of pyquil gates to decompose the Clifford elements into. These must generate the
+         clifford group on the qubits of interest. e.g. for one qubit [RZ(np.pi/2), RX(np.pi/2)].
+        :return: A list of pyquil programs. Each pyquil program is a circuit that represents an element of the
+         Clifford group. When these programs are composed, the resulting Program will be the randomized benchmarking
```

Comment:
Suggest we make an explicit promise here that the composition will be the identity.